src/Catalyst.Modules.Server.Blazor/Components/SendTransactionForm.razor.cs

```diff
@@ -55,36 +46,6 @@ public class SendTransactionFormBase : ComponentBase
         [Inject]
         public ITransactionReceivedEvent TransactionReceivedEvent { get; set; }
 
-        public void HandleValidSubmit()
-        {
-            //TODO: Currently there is no method to actually set the transaction amount
-            var transaction = new TransactionBroadcast {Timestamp = Timestamp.FromDateTime(DateTime.UtcNow)};
-            transaction.PublicEntries.Add(new PublicEntry
-            {
-                Base = new BaseEntry
-                {
-                    SenderPublicKey = PeerSettings.PublicKey.KeyToByteString()
-                }
-            });
-            var signingContext = new SigningContext
-            {
-                NetworkType = PeerSettings.NetworkType,
-                SignatureType = SignatureType.ProtocolPeer
-            };
-
-            var signature = KeySigner.Sign(transaction.ToByteArray(), signingContext);
-            transaction.Signature = new Signature
-            {
-                RawBytes = signature.SignatureBytes.ToByteString(),
-                SigningContext = signingContext
-            };
-            var status = TransactionReceivedEvent
-               .OnTransactionReceived(transaction.ToProtocolMessage(PeerSettings.PeerId,
-                    CorrelationId.GenerateCorrelationId()));
-            JsRuntime.InvokeAsync<object>("window.alert", $"Transaction Status: {status.ToString()}")
-               .ConfigureAwait(false);
-            Model = new SendTransactionModel();
-            StateHasChanged();
-        }
+        public void HandleValidSubmit() { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4872747)